CMakeLists.txt

```diff
@@ -118,22 +109,22 @@ if(BUILD_SHARED_LIBS)
 			ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
 endif()
 
-if(BUILD_STATIC_LIBS)
-	add_library(TinyEXIFstatic STATIC TinyEXIF.cpp TinyEXIF.h)
-	
-	target_link_libraries(TinyEXIFstatic tinyxml2::tinyxml2)
-	set_target_properties(TinyEXIFstatic PROPERTIES
+if(NOT BUILD_SHARED_LIBS)
+	add_library(TinyEXIF STATIC TinyEXIF.cpp TinyEXIF.h)
```

Comment:
Are these changes necessary? could just not depend on `TinyEXIFstatic` directly?